doop.c

```diff
@@ -635,6 +635,40 @@ Perl_do_trans(pTHX_ SV *sv)
     }
 }
 
+/* TODO: add API docs */
+STATIC bool
+S_sv_has_overload_method(pTHX_ SV *sv, int method)
+{
+    HV *stash = NULL;
+    MAGIC *mg = NULL;
+    CV **cvp = NULL;
+    CV *cv = NULL;
```

Comment:
Don't forget that we're C99 now. We don't have to do this. We can just do

```c
HV *stash = SvSTASH(SvRV(sv));
```

on line 652 below, etc etc.